CHANGES.txt

```diff
@@ -5,6 +5,9 @@ Changes for Crate
 Unreleased
 ==========
 
+ - Fix: The expected nodes setting value is validated against the number of
+   data and client nodes instead of all nodes in the cluster.
```

Comment:
maybe something like

.. the recovery expected nodes check incorrectly included client nodes instead of just data and master nodes

?